pyne/compare_data.py

```diff
@@ -0,0 +1,37 @@
+"""this method compare two columns of results and errors, return
+   ture if the two columns have the same results otherwise return
+   false. Also return the list of absolute difference btw these 
+   two columns"""
+def compare(mode,file1,file2):
```

Comment:
As I've said a couple of times, start very small, and make a function that is very robust and will work for all possible cases of two numbers, each with associated statistical errors.

You are jumping ahead to whole files when we don't need that yet.